src/lib/EventDispatcher.ts

```diff
@@ -81,43 +79,35 @@ export default class EventDispatcher extends Disposable implements IEventDispatc
 	 * _Please note: [[IEvent.preventDefault|preventDefault()]] can only be called on
 	 * events that have their [[IEvent.cancelable|cancelable]] property set to true_
 	 */
-	public dispatchEvent(event:IEvent):boolean
-	{
-		if(this.isDisposed())
-		{
+	public dispatchEvent(event: IEvent): boolean {
+		if (this.isDisposed()) {
 			// todo: _log.error("Can't dispatchEvent on a disposed EventDispatcher");
```

Comment:
I think removing the TODO and throwing an error should be sufficient.

Reply:
Indeed, i'll update this!